content/python/encyclopedia/replace-in-python.md

```diff
@@ -0,0 +1,90 @@
+<!--
+---
+Title: "Replace in Python" 
+Subjects:
+  - "Python"
+Tags: # Please first look to use tags that are already listed at author.codecademy.com/browse/tags
+  - "Functions"
+  - "Python"
+Catalog Content: 
+  - https://www.codecademy.com/learn/learn-python-3"
+---
+-->
+
+Use the built in replace() method in Python to modify strings.
+
+## Syntax
+
+`String.replace("thing_to_remove", "thing_to_add", count)`
+
+replace() can be called either directy on a String, or on a variable assigned to a String:
+
+```Python
+var = "x".replace("x", "y")
+print(var)
+# prints(y)
+
+var = "x"
+var = var.replace("x", "y")
+print(var)
+# prints(y)
```

Comment:
hmm i'd change the example codeblock a bit:

```python
var = "hello x".replace("x", "y")
print(var)
# Output: hello y

var = "hello x"
var = var.replace("x", "y")
print(var)
# Output: hello y
```

Reply:
good point! I also changed x to hello world